PocketKit/Sources/ItemWidgetsKit/Views/ItemWidgetsRow.swift

```diff
@@ -18,23 +18,26 @@ struct ItemWidgetsRow: View {
     let domain: String
     let readingTime: String?
     let image: Image?
+    let url: URL
```

Comment:
Naming this `url` can be misleading.
All(?) pocket items have a url, this url in particular is a deeplink with a custom scheme which I would suggest requires more clarity.